lesson-03-passport-token/exercise-08/exercise.md

```diff
@@ -49,11 +49,11 @@ pub type Extension = Option<Metadata>;
 
 # Exercise
 
-1. Create a variable called `mint_msg`. Assign it a `MintMsg` struct and write each of the four members on their own line. Assign the `token_id` struct field the value of the `token_id` string variable. `owner` will be the string version of the `MINTER` constant (created with `to_string`). Set `token_uri` to `None` (since we're using on-chain metadata), and set the `extension` field to `metadata_extension`.
+1. Create a variable called `mint_msg`. Assign it a `MintMsg` struct and write each of the four members on their own line. Assign the `token_id` struct field the value of the `token_id` string variable. The `owner` field will be the string version of the `MINTER` constant (created with `to_string`). Set `token_uri` to `None` (since we're using on-chain metadata), and set the `extension` field to `metadata_extension`.
 2. Create a variable called `execute_msg` and assign it the `Mint` function from `ExecuteMsg`. Don't forget to pass it `mint_msg` but since we need to use `mint_msg` again you'll have to [clone](https://doc.rust-lang.org/std/clone/trait.Clone.html) it.
-3. Call the `execute` function of the `contract` variable. The arguments you need to pass it, and its call format are similar to what you wrote for `contract.instantiate`. It's the last time we're using `info` so you won't need to `clone` it, and use the `execute_msg` you created in step 2 for it's `msg` parameter.
+3. Call the `execute` function of the `contract` variable. The arguments you need to pass it, and its call format are similar to what you wrote for `contract.instantiate`. It's the last time we're using `info` so you won't need to `clone` it. Use the `execute_msg` you created in step 2 for it's `msg` parameter.
```

Comment:
Found another typo number 3:
"Use the `execute_msg` you created in step 2 for it's `msg` parameter."

Should be:
"Use the `execute_msg` you created in step 2 for its `msg` parameter."